python/gtsam_py/python/__init__.py

```diff
@@ -1,9 +1,11 @@
 from .gtsam import *
 
+import numpy as np
```

Comment:
I think it's better to not import numpy in __init__.py so we don't get `gtsam.np`?

Reply:
Ah that's a good point. Though I am confused why the PointX API has changed and not just handled via GTSAM?

Reply:
Point2 and Point3 are now just numpy arrays, so (

Reply:
Huh that's strange. They were numpy arrays in the cython wrapper too, that's what eigency was for :-)

Reply:
Let me see what I can do about it

Reply:
No they are not
```
import gtsam
p=gtsam.Point3(0,0,0)
>>> type(p)
gtsam.gtsam.Point3
>>> p.vector()
array([0., 0., 0.])
```

Reply:
Yeah the typedefing to Vectors is the key here. I have a fix for the original numpy issue though.

Reply:
So I think this change can be reversed? After that we can merge the setup.py changes :)